examples/hf/gpt2/pippy_gpt2.py

```diff
@@ -169,6 +153,15 @@ def run_master(_, args):
     parser.add_argument('--cuda', type=int, default=int(torch.cuda.is_available()))
     parser.add_argument('--record_mem_dumps', type=int, default=0, choices=[0, 1])
     parser.add_argument('--checkpoint', type=int, default=0, choices=[0, 1])
+    parser.add_argument('--rpc_timeout', type=int, default=1800)
+    parser.add_argument('--num_worker_threads', type=int, default=512)
+
+    parser.add_argument('--n_embd', type=int, default=None)
+    parser.add_argument('--n_layer', type=int, default=None)
+    parser.add_argument('--n_head', type=int, default=None)
```

Comment:
nit: would it be possible to move the above three model config arguments to a json config file?
Just thinking of minimizing the diff compared to the original HF example.
For example, we support using a json file with the Translation example:
```
python -u run_translation.py --model_name_or_path t5-3b  --config_name=t5-3b_config.json ...
```
https://github.com/pytorch/tau/blob/main/examples/hf/translation/t5-3b_config.json